commands/status.go

```diff
@@ -102,3 +118,7 @@ var (
 	Status  = new(statusCommand)
 	StatusC = kingpin.Command("status", "Show information about local pmm-agent")
 )
+
+func init() {
+	StatusC.Flag("wait", "Timeout to get success response from pmm-agent").DurationVar(&Status.timeout)
```

Comment:
Timeout looks like a not proper word here. `s/timeout/deadline/g`?
Maybe better deadline? `Time deadline to get a success response.`
@PaulJacobs-percona , could you please suggest a better word for: a maximum time to wait on the response?

Reply:
True that 'timeout' isn't a dictionary word (well, not UK), but it's familiar with command-line junkies and has a generally-accepted meaning. But my Mac dictionary (american english) says "2 (timeout) Computing a cancellation or cessation that automatically occurs when a predefined interval of time has passed without a certain event occurring."